local/local.go

```diff
@@ -120,6 +120,13 @@ func (i *Image) Env(key string) (string, error) {
 	return "", nil
 }
 
+func (i *Image) Entrypoint() ([]string, error) {
+	if i.inspect.Config.Entrypoint == nil {
+		return []string{}, nil
+	}
+	return i.inspect.Config.Entrypoint, nil
+}
```

Comment:
```suggestion
func (i *Image) Entrypoint() ([]string, error) {
	return i.inspect.Config.Entrypoint, nil
}
```
Given the image can technically have either a `nulll` or empty entrypoint I don't think we should translate null to empty and should instead just return the value.